### DISCUSSION TITLE
Updated Installation Guide For Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by illatus** [Ylianst/MeshCentral#5362] [Q&A] [answered]
Hello ,

A friend of mine recommended me to try out Meshcentral. I managed to install it and run it with its default database. 

Iam trying to use it with the mongodb but i cant make it work. Commands from the installation guide are outdated and i cant
figure out how to connect it with a database. If iam to create a database for example in Mysql what tables does it need? or just
making a database is enough?

**Comment by si458** [comment, marked as answer]
it will automatcially create all the tables thats required
also have a look at the new help docs - https://ylianst.github.io/MeshCentral/meshcentral/#mongodb-setup

**Comment by illatus** [comment]
Thank you for your help.!!

**Reply by si458** [reply]
No worries!
We are aware the PDF docs are a little out of date,
that's why we point people to the docs website now!
